docs/content/integrations/powerbi.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: "Dagster & Power BI"
+description: "Represent Power BI semantic models, data sources, reports, and dashboards as assets in Dagster."
+---
+
+# Dagster & Power BI
+
+<ExperimentalCallout />
+
+This guide provides instructions for using Dagster with Power BI.
```

Comment:
Do you plan to flesh these out more in the near future? If not, consider adding a sentence or two here about why using Power BI with Dagster is beneficial.

Reply:
Added a few sentences, but plan to revisit this later on!